src/graph/rg_matrix/rg_matrix.c

```diff
@@ -244,3 +244,59 @@ void RG_Matrix_validateState
 }
 
 
+void RG_Matrix_get
+(
+	RG_Matrix C,
+	GrB_Matrix M
+) {
```

Comment:
I think it would be nicer if we change the signature to:
```
void RG_Matrix_get
(
    GrB_Matrix *M,   // in/output
    RG_Matrix C
) 
```

`M` will allocated within this function, similar to `GrB_Matrix_new`